services/apps/test/apps/containers_test.clj

```diff
@@ -21,6 +21,12 @@
 
```

Comment:
We should make sure jenkins is running these tests for us. Sort of a public note-to-self.